doc/developers.rst

```diff
@@ -21,7 +21,7 @@ first time:
 
 .. code:: bash
 
-  git clone https://github.com/user/pyglmnet
+  git clone https://github.com/Your_User_Name/pyglmnet
```

Comment:
I would put it in <> so that it's clear that it must be replaced. Something like:

```suggestion
  git clone https://github.com/<your-user-name>/pyglmnet
```